tests/Api/Utils/OrderPlacerTrait.php

```diff
@@ -21,67 +21,115 @@
 use Sylius\Bundle\ApiBundle\Command\Checkout\UpdateCart;
 use Sylius\Component\Core\Model\Address;
 use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\OrderPaymentTransitions;
 use Sylius\Component\Core\Repository\OrderRepositoryInterface;
 use Sylius\Component\Order\OrderTransitions;
 use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\HandledStamp;
 use Webmozart\Assert\Assert;
 
 trait OrderPlacerTrait
 {
-    protected function placeOrder(string $tokenValue, string $email = 'sylius@example.com'): void
-    {
-        /** @var MessageBusInterface $commandBus */
-        $commandBus = self::getContainer()->get('sylius.command_bus');
+    private MessageBusInterface $commandBus;
 
-        $pickupCartCommand = new PickupCart($tokenValue, 'en_US');
-        $pickupCartCommand->setChannelCode('WEB');
-        $commandBus->dispatch($pickupCartCommand);
+    private OrderRepositoryInterface $orderRepository;
 
-        $addItemToCartCommand = new AddItemToCart('MUG_BLUE', 3);
-        $addItemToCartCommand->setOrderTokenValue($tokenValue);
-        $commandBus->dispatch($addItemToCartCommand);
+    public function setUp(): void
+    {
+        parent::setUp();
```

Comment:
```suggestion
    final public function setUpOrderPlacer(): void
    {
```
or
```suggestion
    final public function initializeOrderPlacer(): void
    {
```
And then just use it in the setup of the test.
Adding new things to a setup in an actual test would necessitate copying the body of this if left as is.